lib/mayaUsd/ufe/MayaUsdContextOps.cpp

```diff
@@ -551,47 +323,9 @@ Ufe::ContextOps::Items UsdContextOps::getItems(const Ufe::ContextOps::ItemPath&
         items.emplace_back(Ufe::ContextItem::kSeparator);
 #endif
 
-        if (!fIsAGatewayType) {
-            // Working set management (load and unload):
-            const auto itemLabelPairs = _computeLoadAndUnloadItems(prim());
-            for (const auto& itemLabelPair : itemLabelPairs) {
-                items.emplace_back(itemLabelPair.first, itemLabelPair.second);
-            }
+        // Add the items from our base class here
+        items.insert(items.end(), baseItems.begin(), baseItems.end());
```

Comment:
Luckily all the base class menu items were in one block in the menu. Insert them here.